creativecommons/legaldb
# [Fix] Makes Contribute card component visible on smaller screens

## Fixes



- Fixes #231  by @SumaiyaaRq

## Description

This PR makes the `contribute-card` element visible on smaller screen sizes `( < 1024px )`

## Technical details

- Implements a `.wrapper` element to control element order using css flexbox
- Removes the `.is-hidden-touch` class from the `.contribute-card` child element
- Unsets `max-width`, as well as float styles

## Tests

1. Run the development server
2. Navigate to the cases page
3. Toggle the developer tools for your browser and reduce screen to any size lesser that 1024px
4. Observe the `.contribute-card` component does not dissapear
5. Confirm that the `.contribute-card` component is responsive all the way to 320px
6. Confirm that no build or runtime errors occur as a result of the changes.

## Screenshots

![image](https://github.com/user-attachments/assets/f7b4ed32-7b90-499e-9879-9e86f9c08a36)


## Checklist



  index.md`).
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin




For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
